Lib/test/test_strptime.py

```diff
@@ -370,22 +372,39 @@ def test_offset(self):
         (*_, offset), _, offset_fraction = _strptime._strptime("-013030.000001", "%z")
         self.assertEqual(offset, -(one_hour + half_hour + half_minute))
         self.assertEqual(offset_fraction, -1)
-        (*_, offset), _, offset_fraction = _strptime._strptime("+01:00", "%z")
+
+    @parameterize(
```

Comment:
This test LGTM. :) Yet, it would also be nice to extend (appropriately) the relevant tests in `Lib/test/datetimetester.py` (regarding both implementations of the `datetime` module):

* `TestDateTime`: `test_strptime()`
* `TestTime`:
  * `test_strptime_tz()`
  * `test_strptime_errors()`